docs/tutorial/useful Code-Snippets.txt

```diff
@@ -0,0 +1,72 @@
+http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup
```

Comment:
I think this file is not really required

Reply:
that was just for myself. I forgot to add it to .gitignore ^^